pipeline.py

```diff
@@ -70,20 +70,36 @@
     description="InstructLab pipeline",
 )
 def ilab_pipeline(
+    # Model I/O
+    input_model_uri: str,
+    output_oci_model_uri: str = None,
```

Comment:
A slight implementation detail here is that `dsl.If` doesn't support `None` as a check, so we'll likely need to do something like this:
```suggestion
    output_oci_model_uri: str = "",
```